packages/algob/src/builtin-tasks/test.ts

```diff
@@ -6,10 +7,17 @@ import type { Config } from "../types";
 import { loadFilenames } from "./deploy";
 import { TASK_TEST } from "./task-names";
 
+const TEST_DIR = 'test';
 async function runTests (config: Config): Promise<void> {
   try {
-    const testFiles = loadFilenames(testsDirectory, "test");
-    console.log("Test files:", testFiles);
+    const tsPath = findupSync("tsconfig.json", { cwd: process.cwd() });
+    if (tsPath) {
+      // run tests via ts-mocha, if project is in typescript
+      process.env.TS_NODE_PROJECT = tsPath;
+      require('ts-mocha');
```

Comment:
good, this should not be imported in a non typescript project because it's a big dependency and takes lot of time.

Reply:
yes